test_cli_integration.py

```
@@ -0,0 +1,147 @@
+            mock_instance.config_list = [{'model': 'gpt-4'}]
+            mock_handle.return_value = "Test response"
+            
+            # Test with --continue flag
+            try:
+                _run_prompt(
+                    "Test prompt",
+                    continue_session=True,
+                    no_save=False
+                )
+                print("✅ --continue flag handled successfully")
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)